_posts/2020-05-18-First-Trial-Data-Uploads.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: blog
+author: Tony Lorenzo
+---
+
+This post describes the first test trial of the Solar Forecast Arbiter
+framework, and a script that participants can uses to generate and post
+random forecasts for the trial.
+
+To set up a trial, the framework administrators perform the following steps:
+
+1. Work with stakeholders to define trial parameters
+2. Create anonymous users
+3. Create Sites and Observations (as required)
+4. For each anonymous user, create the Forecast objects
+5. Create periodic and final Reports of how the forecasts performed
+
+Trial participants will receive an email with their unique, anonymous
+username to use specifically for the trial along with a link to set a
+password.  Participants then use this trial username and password to
+upload forecast values for each forecast object assigned to them.  In
+most cases, the framework will restrict uploads so only those made
+before the forecast issue time of day are valid.
+
+**Example Script**
+
+An example script to upload random forecast values for each of the user's
+forecast objects in a trial can be found
+[here](https://solarforecastarbiter.org/assets/test_trial_1_random_forecast_upload.py).
```

Comment:
hmmm I'm not opposed to just dumping it in `assets` but it seems like there should be a better place. A new repository would be fine with me. If you think that's too heavy then maybe a gist?

Reply:
Repository is a little heavy but a gist is a great idea

Reply:
Has the script been tested from outside of U of A?  If not, would you like me to find a vendor willing to do that?

Reply:
@MrWindAndSolar the script is to give them an idea/example of how to program against the API. We expect most vendors will want to run their own scripts. So I'm not super concerned that it's robust to many user environments/installations. But let us know if you feel otherwise from your conversations with vendors.